Resources/config/schema/create-1.0.xsd

```diff
@@ -58,6 +59,11 @@
         <xsd:attribute name="manager-name" type="xsd:string" />
     </xsd:complexType>
 
+    <xsd:complexType name="orm">
+        <xsd:attribute name="enabled" type="xsd:string" />
+        <xsd:attribute name="manager-name" type="xsd:string" />
+    </xsd:complexType>
```

Comment:
@WouterJ may i ask you to sanity-check if this is correct? i think it is, but in the past i have been remarkably bad at spotting errors in xml schemas...